app/controllers/hostgroups_controller.rb

```diff
@@ -103,6 +112,10 @@ def process_hostgroup
     render :partial => "form"
   end
 
+  def csv_columns
+    [:name, :hosts_count, :children_hosts_count]
```

Comment:
`:name` only shows the deepest nested hg name, this should use `:title` to get the full name